source/leuven/superdome_quick_start.rst

```diff
@@ -1,77 +1,42 @@
 Superdome quick start guide
 ===========================
 
-The :ref:`Superdome <Superdome hardware>` is a shared memory machine, i.e.,
-it can be used to run multi-threaded application that require large amounts
-of RAM.
-
-
-How to connect to Superdome?
-----------------------------
-
-Superdome does not have a dedicated login node, so in order to work with the
-system users can connect to the same login node used for Genius.
```

Comment:
Just for the sake of completeness, please provide a link to the Genius login node reference page with the anchor `tier2_login_nodes`.

Reply:
Yes, I see that I somehow moved that part to the Superdome hardware page instead of here, I'll put it back (but just a reference to the Genius login infopage, no need to repeat the whole thing).

Reply:
done (34d94295f9fc4bbc8a30f23b80fdc2e510192267)

Reply:
exactly. that will do